app/lib/screens/wallets/send.dart

```diff
@@ -374,22 +376,60 @@ class _WalletSendScreenState extends ConsumerState<WalletSendScreen> {
             MaterialPageRoute(builder: (context) => const ScanScreen()));
       }
     }
-    if (result.rawValue != null) {
-      final code = Uri.parse(result.rawValue!);
+    if (result.rawValue != null && result.rawValue!.startsWith('TFT')) {

```

Comment:
don't check if it starts with `TFT`, you can check after parsing the scheme -> `code.scheme`